pkg/monitoring/metrics/virt-controller/vmstats_collector.go

```diff
@@ -560,3 +569,19 @@ func collectDiskMetricsFromPVC(vm *k6tv1.VirtualMachine) []operatormetrics.Colle
 
 	return cr
 }
+
+func CollectVMCreatedTime(vms []*k6tv1.VirtualMachine) []operatormetrics.CollectorResult {
+	var cr []operatormetrics.CollectorResult
+
+	for _, vm := range vms {
+		if !vm.CreationTimestamp.IsZero() {
```

Comment:
I'd prefer early exit :)  But fine.
Question: Do we care about deleted VMs?
If not, maybe we should skip those with `deletionTimestamp != nil`

Reply:
we want to see this info also for deleted vms- @sradco right?

Reply:
No, We don't report metrics for deleted VMs.

Reply:
sure, so fixed @fossedihelm

Reply:
But I think teh deleted date is there as an indication that the VM was asked to be deleted. Not that its already deleted.
Am I wrong?

Reply:
No, that should be to avoid adding something that "it's gonna to be deleted"

Reply:
We dont do this check for ant other metric and I dont think we should start.
There are cases where a VM has a delete date but for some reason it is stuck. We should still report the metric until the resource doesnt exists anymore.

Reply:
so I fixed to code back to include it (:

Reply:
cc @fossedihelm

Reply:
Fine! Thank you for the explanation